tests/common.py

```diff
@@ -122,14 +122,27 @@ def wait_for_inputs_delete(stub, input_ids, metadata):
         raise Exception(f"Timeout after {timeout} seconds to delete inputs {remaining_input_ids}")
 
 
-def wait_for_model_trained(stub, metadata, model_id, model_version_id, user_app_id=None):
+def wait_for_model_trained(
+    stub, metadata, model_id, model_version_id, user_app_id=None, retry_on_internal_failure=False
+):
+    retry_count = 0
     while True:
         response = stub.GetModelVersion(
             service_pb2.GetModelVersionRequest(
                 user_app_id=user_app_id, model_id=model_id, version_id=model_version_id
             ),
             metadata=metadata,
         )
+        ## Temp fix for EAGLE-4649 until LT-5281 is implemented.
```

Comment:
note: this comment did result in automatically closing LT-5281 when this PR was merged. I did re-open the ticket and we will work on it next Sprint.